src/lib/addons/teams.ts

```diff
@@ -23,6 +41,26 @@ export default class TeamsAddon extends Addon {
     flagResolver: IFlagResolver;
 
     constructor(args: IAddonConfig) {
+        if (args.flagResolver.isEnabled('teamsIntegrationChangeRequests')) {
+            teamsDefinition.events = [
```

Comment:
This is the only place the teamsDefinition is used yeah?

Reply:
Just double checked: Yes.